### DISCUSSION TITLE
Problem no input response html to dom

### DISCUSSION THREAD

**Discussion by bubixon** [livewire/livewire#6017] [1. Help]
Error when i  click wire:click all on backend working but livewire not put html response to dom
<img width="891" alt="image" src="https://github.com/livewire/livewire/assets/41482154/d4290ee7-eab7-42a8-aa18-d79fb0770e68">

I have a problem I have a tabs component everything on the responsa side etc works changes the model etc but it gives me this error response is correct but does not update the appearance returned by livewire on the frontend

<img width="670" alt="image" src="https://github.com/livewire/livewire/assets/41482154/36b68eaa-7cf2-4e93-8443-baeae536d14a">

**Comment by danie-ramdhani** [comment]
duplicate #5995

**Reply by bubixon** [reply]
Thanks
